### DISCUSSION TITLE
Pkl Language Server 0.1.2 is released

### DISCUSSION THREAD

**Discussion by stackoverflow** [apple/pkl#835] [Announcements]
Version 0.1.2 is released with support for Pkl `0.27`.

See the full [changelog](https://pkl-lang.org/lsp/current/CHANGELOG.html#release-0.1.2).

**Comment by vincentvdk** [comment]
Any idea when native binaries will become available?

**Reply by bioball** [reply]
Currently, this is blocked by https://github.com/oracle/graal/issues/8113. Once that is ready, we should be able to publish pkl-lsp as a native binary.
